pyne/ensdf_processing.py

```diff
@@ -104,3 +104,105 @@ def gabs(inputdict_unchecked):
     proc.communicate()[0]
     proc.stdin.close()
 
+def logft(inputdict_unchecked):
+    #NOTE: changed input file line length to 90 to support longer file paths in fortran source.
+    """
+    This program calculates log ft values for beta and electron-capture decay, average beta energies, 
+    and capture fractions.  (LOGFT readme)
+
+    Input Dictionary Required Key Pair Value:
+        input_data_set : path to input data file.
+        output_report : desired path to output report file.
+        data_table : path to data table.
+        output_data_set : desired path to output data set.
+
+    Output Dictionary Values:
+        Everything in input dictionary is returned if LOGFT completes successfully.
+    """
+    inputdict = {}
+    input_data_set = inputdict_unchecked['input_data_set']
+    output_report = inputdict_unchecked['output_report']
+    data_table = inputdict_unchecked['data_table']
+    output_data_set = inputdict_unchecked['output_data_set']
+
+    exe_path = path_to_exe('logft')
+    inp = input_data_set + '\n' + output_report + '\n' + data_table + '\n' + output_data_set + '\n'
+    proc = subprocess.Popen([exe_path],stdout=subprocess.PIPE,stdin=subprocess.PIPE)
```

Comment:
@scopatz The executable ask for a bunch of info via command line, but they can't be passed in as arguments.  I thought I had to use Popen to be able to communicate with the executable?  I also saw the following on:
```
Note Do not use stderr=PIPE with this function as that can deadlock based on the child process error volume. Use Popen with the communicate() method when you need a stderr pipe.
```
https://docs.python.org/2/library/subprocess.html
Do you know of a good way for me to communicate with the executable using subprocess.check_output()?  Sorry if this response is particularly naive, I don't have much experience working with subprocess.